internal/binutils/binutils.go

```diff
@@ -159,12 +243,24 @@ func findExe(cmd string, paths []string) (string, bool) {
 // of a binary.
 func (bu *Binutils) Disasm(file string, start, end uint64, intelSyntax bool) ([]plugin.Inst, error) {
 	b := bu.get()
-	args := []string{"-d", "-C", "--no-show-raw-insn", "-l",
-		fmt.Sprintf("--start-address=%#x", start),
-		fmt.Sprintf("--stop-address=%#x", end)}
+	if !b.objdumpFound && !b.llvmObjdumpFound {
+		return nil, fmt.Errorf("couldn't find objdump")
+	}
+	var args []string
+
+	if b.llvmObjdumpFound {
+		args = []string{"--disassemble-all", "--no-show-raw-insn",
+			"--line-numbers", fmt.Sprintf("--start-address=%#x", start),
```

Comment:
Either --line-numbers is -l is accepted by all these objdump programs. Unify the option names.

Reply:
llvm-objdump (Apple LLVM objdump) should support `-C`. Is it not the case on your machine?

Reply:
Yes, looks like they can be unified. I will verify the flags and combine.

Reply:
Done.